csrankings.csv

```diff
@@ -9002,6 +9003,7 @@ Manuel Carro,IMDEA Software Institute,http://software.imdea.org/people/manuel.ca
 Manuel Chakravarty,UNSW,http://www.cse.unsw.edu.au/~chak,w6XYY_EAAAAJ
 Manuel E. Acacio,University of Murcia,http://www.ditec.um.es/~meacacio,iHD6MsUAAAAJ
 Manuel E. Bermudez,University of Florida,http://www.cise.ufl.edu/~manuel,xjUqBT0AAAAJ
+Manuel E. Lladser,University of Colorado Boulder,http://amath.colorado.edu/faculty/lladser/,Uv6cAEMAAAAJ

```

Comment:
Not listed here: https://www.colorado.edu/cs/faculty and does not mention an appointment with CS. Please clarify how he meets the inclusion criteria.

Reply:
That page is unfortunately incomplete. The affiliation is stated here: https://experts.colorado.edu/display/fisid_134170